docs/getting-started/app-custom-theme.md

```diff
@@ -234,3 +234,73 @@ const LogoFull = () => {
   return <img src={MyCustomLogoFull} />;
 };
 ```
+
+## Custom Homepage
+
+In addition to a custom theme, a custom logo, you can also customize the
+homepage of your app. To do that we need to go through a few steps.
+
+### Setting up the Home Page
+
+1. Create a Home Page Component that will be used for composition.
+
+`packages/app/src/components/home/HomePage.tsx`
+
+```tsx
+import React from 'react';
+
+export const HomePage = () => {
+  return {
+    /* TODO: Compose a Home Page here */
+  };
+};
+```
+
+2. Add a route where the homepage will live, presumably `/`.
+
+`packages/app/src/App.tsx`
+
+```tsx
+import { HomepageCompositionRoot } from '@backstage/plugin-home';
+import { HomePage } from './components/home/HomePage';
+
+// ...
+<Route path="/" element={<HomepageCompositionRoot />}>
+  <HomePage />
+</Route>;
+// ...
+```
+
+### Composing your Home Page
+
+Composing a Home Page is no different from creating a regular React Component,
+i.e. the App Integrator is free to include whatever content they like. However,
+there are components developed with the Home Page in mind. If you are looking
+for components to use when composing your homepage, you can take a look at the
+[collection of Homepage components](https://backstage.io/?path=/story/plugins-home-components)
+in storybook. If you don't find a component that suits your needs but want to
+contribute, check the
+[Contributing documentation](https://github.com/backstage/backstage/blob/1fa18f2ab60abf534301bcf2b47c63b49615cf93/plugins/home/README.md#contributing).
```

Comment:
this link and the one below probably should not be to hashes, but rather to `master`, so that readers will see future updates to the readme, right?